benchmarks-next/blake3-scalar/rust-benchmark/Cargo.toml

```diff
@@ -5,5 +5,6 @@ edition = "2018"
 
 [dependencies]
 blake3 = "0.3.7"
+sightglass-api = { path = "../../../crates/api" }
```

Comment:
I think we would probably have to publish the sightglass-api crate, not use a local path, since these benchmarks are built inside Docker.